diff --git a/node_modules/@lerna-lite/core/dist/command.js b/node_modules/@lerna-lite/core/dist/command.js
index ad38175..068accc 100644
--- a/node_modules/@lerna-lite/core/dist/command.js
+++ b/node_modules/@lerna-lite/core/dist/command.js
@@ -184,7 +184,8 @@ export class Command {
         if (this.commandName !== 'info') {
             chain = chain.then(() => this.project.getPackages());
             chain = chain.then((packages) => {
-                this.packageGraph = new PackageGraph(packages || [], 'allDependencies');
+                const { graphType } = this.options.command?.[this.commandName] ?? {};
+                this.packageGraph = new PackageGraph(packages || [], graphType ?? 'allDependencies');
             });
         }
         return chain;